docs/configure_https.md

```diff
@@ -1,8 +1,10 @@
 # Configuring Harbor with HTTPS Access
 
-Because Harbor does not ship with any certificates, it uses HTTP by default to serve registry requests.  However, it is highly recommended that security be enabled for any production environment. Harbor has an Nginx instance as a reverse proxy for all services, you can use the prepare script to configure Nginx to enable https.
+Because Harbor does not ship with any certificates, it uses HTTP by default to serve registry requests. However, using HTTP is acceptable only in air-gapped test or development environments that do not have a connection to the external internet. Using HTTP in environments that are not air-gapped exposes you to man-in-the-middle attacks. In production environments, always use HTTPS. If you enable Content Trust with Notary, you must use HTTPS. 
```

Comment:
maybe change to 'in production environments where use should use Content Trust with Notary to properly sign all images, you must use HTTPS.'

Reply:
@stuclem  Hi Stuart, I just discussed with @reasonerjt and @ninjadq  that we are making https the default (in harbor yml) but current users of http will not be impacted when they upgrade to new release ie nothing will be break and they still experience http as the default, instead we will force a cutover on a future release after proper announcement.  But basically all first time users will be using https as default mode.

Reply:
Did I miss anything else? Is there anything else that Stuart should be notified of from a doc POV? @reasonerjt

Reply:
@xaleeks  I added the following in https://github.com/goharbor/harbor/pull/9508/commits/1159a2785138c5855fb36fff35d02cc84d1116ec: 

"In production environments, always use HTTPS. If you enable Content Trust with Notary to properly sign all images, you must use HTTPS."

I want to keep this as two sentences, otherwise it gives the impression that production environments without Notary can live without HTTPS.

Reply:
@xaleeks and @reasonerjt if the default is changing to HTTPS in the next release, should I leave the doc as it is for now? I am hoping to merge my updates to the install guide today or tomorrow. So, should we leave the 1.9.x implementation (i.e. HTTP default) in the doc for the time being? Thanks!